product_docs/docs/transporter/ba_preview/config_reader.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "Configuring"
+---
+
+## Config a source reader
+
+### Get credentials
+
+- Create and download a migration credentials from the postgres AI platform Migrate -> Credentials section
```

Comment:
I could not find an option to create and download credentials. Here is where I went looking: 

With my EDB employee Okta account, I logged into EDB Postgres AI. I selected Projects > chose a Project. On the left navigation bar, I selected Migrate > Migrations. 

Is this the right place to look? Or is there a staging/testing environment you could give @djw-m  and myself access to?

Reply:
so far is not open in production, only in dev/staging. But if you don't have access to those environments I can enable it for you in production.

Dev: https://portal.enterprisedb.network/
Staging: https://portal.staging.enterprisedb.network/

Reply:
In dev, I can log in, but see no projects, so I assume I might need permissions. 
But in staging, I can see the credentials tab 🥳 . 
Thanks!

Reply:
I do now, thanks for the access!